### ISSUE TITLE
Fine-Tuning specifications for MNLI/XNLI

### ISSUE THREAD

**Issue by SinghJasdeep** [google-research/bert#328]
Hey! What batch size, learning rate, max_seq_length, # of epochs, etc. was used to achieve the reported MNLI & XNLI results? 

Thank you.

**Comment by student-06** [comment]
Hey @SinghJasdeep. Just a side question, were you able to run BERT classifier for MNLI dataset ? I was able to run it for MRPC, but for any classification task involving more than 2  classes I get size mismatch error.

Thanks.

**Comment by eric-haibin-lin** [comment]
@SinghJasdeep were you able to find out the hyper-params for MNLI?

**Comment by ijkilchenko** [comment]
I am not sure what exact hyperparameters were used to achieve the reported MNLI results, but if you want to run the classification, you want to switch the task name and the data dir so you should play around with something like this: 

`
python run_classifier.py   --task_name=MNLI   --do_train=true   --do_eval=true   --data_dir=$GLUE_DIR/MNLI   --vocab_file=$BERT_BASE_DIR/vocab.txt   --bert_config_file=$BERT_BASE_DIR/bert_config.json   --init_checkpoint=$BERT_BASE_DIR/bert_model.ckpt   --max_seq_length=128   --train_batch_size=32   --learning_rate=2e-5   --num_train_epochs=3.0   --output_dir=/tmp/mnli_output/
`
